### ISSUE TITLE
Online Portal System for Jan Dhan Yojana (PMJDY)

### ISSUE THREAD

**Issue by KetanArwat** [KetanArwat/Jan-Dhan-Yojana-PMJDY-#1]
System will accept account related information(KYC Documents), from the user to open the account. System will act as a portal between user and the banks. After the KYC documents are uploaded on the system, the user will have to submit other information and choices for their account. Once done, this data will be passed to the respective bank's database. There it will be verified and passed on to PMJDY office for finalizing the account creation.
	Once the account is created the PMJDY office will sent a confirmation message to the account-holder as well as the system. This will significantly reduce the work load from the banks for the PMJDY related work.

**Benefits:**
Reducing work load of the banks. Costumer has to will not have to visit bank for account creation, using the system as the portal an individual’s account can be created from anywhere.
Information submission is done by user at their own time, no need to wait for bank timings for submissions.
Reduced time for overall procedure of account creation.
[ESDL Assignment 5.docx](https://github.com/KetanArwat/Jan-Dhan-Yojana-PMJDY-/files/502205/ESDL.Assignment.5.docx)

**Comment by metro-planning-in-pune-city** [comment]
good information .......!!!!!!!!!!!!!!!!

**Comment by Snehal-Pooja-Shital** [comment]
Nice Idea.....Good Job!!!....But is this Yojna valid for international level.

**Comment by truptisuryavanshi** [comment]
Good Job....Nice Idea.....

**Comment by vivek-omkar-amey** [comment]
Good Project and is benificial

**Comment by ChetanRajput** [comment]
Definitely If implemented, this idea will reduce the hectic.
Nice One...!!

**Comment by KetanArwat** [comment]
@Snehal-Pooja-Shital 
No, the scheme itself is for National Level coverage. So our system is for the same.

**Comment by mamtapanda26** [comment]
Nice idea Ketan and Group...

**Comment by poojasonu** [comment]
Great job ketan and Group....
